src/Expander.hs

```diff
@@ -111,7 +111,8 @@ expandModule thisMod src = do
   modifyState $ set expanderCurrentBindingTable mempty
   startDefTypes <- view expanderDefTypes <$> getState
   local (set (expanderLocal . expanderModuleName) thisMod) do
-    lang <- mustBeModName (view moduleLanguage src)
+    kPath <- klisterPath
+    lang <- mustBeModName kPath (view moduleLanguage src)
```

Comment:
Shouldn't `mustBeModName` be doing the `klisterPath` lookup? That'll prevent repeated code at each call site, and I don't expect that each caller has an interest in varying the path.

Reply:
Sure, good idea!